### ISSUE TITLE
Eq and compression with automation gain compensation 

### ISSUE THREAD

**Issue by bsjdsp** [sadko4u/lsp-plugins#188]
when adjusting the eq the gain output will automatically be gain staged to the level before it was eq adjusted.

**Comment by unsaph3** [comment]
I was going to go ask for this feature, but yes I agree - it would be a godsend to be able to have a switch - "Gain Compensation" that linked the two gains so that the more the input gain is turned up, the more the output gain is lowered to compensate. De-selecting that switch would allow power users to jack output gain as needed.

**Comment by sadko4u** [comment]
This feature won't be implemented earlier than in 1.2.0

**Comment by bsjdsp** [comment]
So you mean you have plans to include it in your plugins

**Comment by sadko4u** [comment]
Yes, that could be a good feature but the actual date of implementation can not be established at this moment.

**Comment by bsjdsp** [comment]
Right I understand. What’s the latest version number from 1.20

**Comment by sadko4u** [comment]
Please don't confuse between 1.1.20 and 1.2.0.
The recent release version is 1.1.30.

**Comment by bsjdsp** [comment]
Oh ok thanks for clarifying
